spec/support/stub_helpers.rb

```diff
@@ -0,0 +1,6 @@
+def stub_delivery_locations
```

Comment:
I like the addition of this file. This project has many stubbed interactions littered about the test files. In retrospect if probably makes them way longer than need to be and kind of hard to read. Would moving them over to this file with a corresponding method for each stub be helpful?

Reply:
Sounds like it could make things more readable. I haven't spent much time in the other tests, but just from getting a brief sense of the number of external calls that seem to be used in this codebase, seems like it could be nice!